### DISCUSSION TITLE
Is there a standard syntax for the comment?

### DISCUSSION THREAD

**Discussion by celtic-coder** [pml-lang/pml-userman#8] [General]
Hi Christian (@pml-lang),

I noticed in the following PML files for the User Manual, that the *comments* were `[--` and `--]` rather than the single `[-` and `-]`.

- 03_01_document_tree_example.pml

https://github.com/pml-lang/user-manual/blob/f764dc6a0c7455381564ec2bcfa572186dc3ab6b/input/text/03_01_document_tree_example.pml#L2-L4

- 07_01_comments.pml

https://github.com/pml-lang/user-manual/blob/f764dc6a0c7455381564ec2bcfa572186dc3ab6b/input/text/07_01_comments.pml#L10-L19

- 11_customization.pml

https://github.com/pml-lang/user-manual/blob/f764dc6a0c7455381564ec2bcfa572186dc3ab6b/input/text/11_customization.pml#L100-L106

Is this an older syntax? The Reference Manual in the *Comments* section under [Text Processing Nodes](https://www.pml-lang.dev/docs/reference_manual/index.html#ch__11), states that the comment has a single `[-` and `-]`, although earlier in the same Reference Manual, in the definition of [Space Character](https://www.pml-lang.dev/docs/reference_manual/index.html#node_sp), it has a reference to a comment using the double dash: `[-- 2 spaces will result in ONE space --]`.

I realize that the *single dash* version encompasses the *double dash* version so that it doesn't matter at the end of the day. For example, from what I can see, Sublime PML highlights each one correctly. 

However, is one or the other the preferred or standard version? And if this is the case, should the User Manual and the Reference Manual consistently reflect this standard?

Kind Regards,
Liam

**Comment by pml-lang** [comment]
Good catch! Thank you.

> Is this an older syntax?

Yes, indeed. I changed the syntax from `[-- --]` to `[- -]`, but forgot some updates.

The online [User Manual](https://www.pml-lang.dev/docs/user_manual/index.html#ch__9) is now fixed, and the Github [repo](https://github.com/pml-lang/user-manual) is updated.

Chapter [Space Character](https://www.pml-lang.dev/docs/reference_manual/index.html#node_sp) in the online reference manual is fixed too.
